nservicebus/sagas/timeouts.md

```diff
@@ -10,15 +10,15 @@ related:
 
 Assumptions can not be made in a message-driven environment regarding the order of received messages and exactly when they will arrive. While the connection-less nature of messaging prevents a system from consuming resources while waiting, there is usually an upper limit to a waiting period that the business dictates. 
 
-The upper wait time is modeled in NServiceBus as a `Timeout`:
+The upper wait time is modelled in NServiceBus as a `Timeout`:
```

Comment:
Our official company language is US English (😢) so the [current spelling is correct](https://www.scribbr.com/us-vs-uk/modelling-vs-modeling/).

```suggestion
The upper wait time is modeled in NServiceBus as a `Timeout`:
```